lua/refactoring/lsp_utils.lua

```diff
@@ -16,7 +17,7 @@ end
 
 local function to_region(pointOrRegion)
     if not pointOrRegion.end_row then
-        return pointOrRegion:to_region()
+        return Region:from_point(pointOrRegion)
```

Comment:
lets rename `from_point` to `to_region` because its a region or a point, not a just a point.

Reply:
I am not sure what you mean here, the caller function is already named `to_region`. I suggest just using Region:from_values here, there is no other place where a point needs to be turned into a region (maybe we should try to adapt the caller so that we don't need to check here whether the arg is a point or a region in the first place)

Reply:
Actually I just checked the code, the call stack is 
```debug/printf.lua -> lsp_utils.insert_new_line_text -> to_region``` or
```lsp_utils_spec.lua -> lsp_utils.insert_test -> to_region```

However, both calls already pass a region. So at the moment the check is unnecessary. I think the easiest is just to have the callers passing regions and never points (there is an easy way of turning points into regions so this won't block future functionality). At the moment there are no other callers of this methods, so the decision can still be taken.

Reply:
Using Region:from_values at this place didn't add any value compared to the current state, it just added unnecessary extra lines

Reply:
this seems fine then.

There is this small problem where a region exists within a text editor.  The problem becomes that currently a point doesn't, meaning there is no bufnr associated with a Point.  Something to think about for future me.